### PR TITLE
Update the demo to use Jenkins 2.240 and recent plugins

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Just an initial cleanup of the demo. I will update the demo to use the published plugins in a follow-up

### CODE REVIEW

File: demo/enable-theme.groovy [hunk 1]
```diff
@@ -24,31 +24,11 @@ if (!darkLogin) {
 
 // Login Theme
 def loginThemeConfig = Jenkins.instance.getExtensionList(SimplePageDecorator.class).get(LoginTheme.class)
```
Comment (timja) [inline demo/enable-theme.groovy:26]: is this needed anymore? its built in now?

File: demo/enable-theme.groovy [hunk 2]
```diff
@@ -24,31 +24,11 @@ if (!darkLogin) {
...
-}
-
+loginThemeConfig.useDefaultTheme = true
 if (!disabled) {
     if (developerMode) {
-        loginThemeConfig.head += '<link rel="stylesheet" href="http://localhost:8081/theme.css" type="text/css" />'
+        loginThemeConfig.head = '<link rel="stylesheet" href="http://localhost:8081/theme.css" type="text/css" />'
     } else {
-        loginThemeConfig.head += '<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jenkinsci/dark-theme@master/src/main/resources/io/jenkins/plugins/darktheme/theme.css" type="text/css" />'
+        loginThemeConfig.head = '<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jenkinsci/dark-theme@master/src/main/resources/io/jenkins/plugins/darktheme/theme.css" type="text/css" />'
```
Comment (timja) [inline demo/enable-theme.groovy:32]: master doesn't seem to work very well as it seems to be cached for a very long time, using a tag might be safer?

File: demo/plugins.txt [hunk 3]
```diff
@@ -1,2 +1,6 @@
+# Development mode
 simple-theme-plugin:0.6
 login-theme:1.1
+# Common mode
+dark-theme:0.0.3
```
Comment (timja) [inline demo/plugins.txt:5]: I'm planning to add something to make development easier,
(needed for the blue ocean work :p )

Comment (oleg-nenashev) [reply demo/plugins.txt:5]: Yes. I am basically revering the login theme magic I was using for development in this demo

File: demo/enable-theme.groovy (hunk 1 above)
Comment (oleg-nenashev) [reply demo/enable-theme.groovy:26]: > its built in now?

Once we switch to the plugin, yes

**Comment by timja** [comment]
🚢 @oleg-nenashev ?

**Comment by oleg-nenashev** [comment]
I will do another update later

**Pull request merged**